pkg/resources/strings.go

```diff
@@ -27,7 +27,6 @@ func ShortenString(s string, n int) string {
 	cutSize := n - postfixLen
 
 	if n < (hashLen + 1) {
-		n = len(s)
```

Comment:
variable 'n' isn't used so it can be removed

Reply:
The line `n = len(s)` is important for handling very small n values, ensuring meaningful string shortening. Despite not being directly used in subsequent lines, it might affect the function's logic indirectly. Removing it without consideration could lead to incorrect behaviour. Imo it wasn't part of your ticket, so you shouldn't remove code which is not related to your jira. Correct me if I'm wrong.

Reply:
It was part of Jira. Flagged as follows by SA4006: `pkg/resources/strings.go:30:3  staticcheck  SA4006: this value of n is never used`